app/controllers/posts_controller.rb

```diff
@@ -42,22 +38,21 @@ def update
 
     else
       redirect_to posts_path
```

Comment:
If the update fails you might want to redirect to the edit path and allow the user to try again... depending on the reason for said failure.  What do you think?